gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -237,10 +270,25 @@ Dump::visit (BlockExpr &block_expr)
 {
   stream << "BlockExpr: [";
   indentation.increment ();
-  stream << std::endl;
+  stream << "\n";
+
   // TODO: inner attributes
+  stream << indentation;
+  stream << "inner attributes: ";
+  if (!block_expr.inner_attrs.empty ())
+    {
+      for (auto &attr : block_expr.inner_attrs)
+	{
+	  stream << "\n";
+	  stream << indentation;
+	  // stream << attr.as_string ();
```

Comment:
Once you are doing the final cleanup, do not leave dead-code behind. ;)

Reply:
Finished